contracts/scripts/wait-for-contract-deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Was getting a few errors as the aggregator was not waiting for the contracts to deploy. See [example errors in this workflow](https://github.com/web3well/bls-wallet/actions/runs/4139071494/jobs/7156223194). 

Andrew suggested creating a similar script to `wait-for-rpc.sh` to overcome this, and this was the solution.

Following code was actually courtesy of chatGPT.